packages/roosterjs-editor-plugins/lib/plugins/Picker/PickerPlugin.ts

```diff
@@ -306,29 +309,31 @@ export default class PickerPlugin<T extends PickerDataProvider = PickerDataProvi
         if (this.isSuggesting) {
             // Word before cursor represents the text prior to the cursor, up to and including the trigger symbol.
             const wordBeforeCursor = this.getWord(event);
-            const wordBeforeCursorWithoutTriggerChar = wordBeforeCursor.substring(1);
-            const trimmedWordBeforeCursor = wordBeforeCursorWithoutTriggerChar.trim();
-
-            // If we hit a case where wordBeforeCursor is just the trigger character,
-            // that means we've gotten a onKeyUp event right after it's been typed.
-            // Otherwise, update the query string when:
-            // 1. There's an actual value
-            // 2. That actual value isn't just pure whitespace
-            // 3. That actual value isn't more than 4 words long (at which point we assume the person kept typing)
-            // Otherwise, we want to dismiss the picker plugin's UX.
-            if (
-                wordBeforeCursor == this.pickerOptions.triggerCharacter ||
-                (trimmedWordBeforeCursor &&
-                    trimmedWordBeforeCursor.length > 0 &&
-                    trimmedWordBeforeCursor.split(' ').length <= 4)
-            ) {
-                this.dataProvider.queryStringUpdated(
-                    trimmedWordBeforeCursor,
-                    wordBeforeCursorWithoutTriggerChar == trimmedWordBeforeCursor
-                );
-                this.setLastKnownRange(this.editor.getSelectionRange());
-            } else {
-                this.setIsSuggesting(false);
+            if (wordBeforeCursor) {
+                const wordBeforeCursorWithoutTriggerChar = wordBeforeCursor.substring(1);
+                const trimmedWordBeforeCursor = wordBeforeCursorWithoutTriggerChar.trim();
+
+                // If we hit a case where wordBeforeCursor is just the trigger character,
+                // that means we've gotten a onKeyUp event right after it's been typed.
+                // Otherwise, update the query string when:
+                // 1. There's an actual value
+                // 2. That actual value isn't just pure whitespace
+                // 3. That actual value isn't more than 4 words long (at which point we assume the person kept typing)
+                // Otherwise, we want to dismiss the picker plugin's UX.
+                if (
+                    wordBeforeCursor == this.pickerOptions.triggerCharacter ||
+                    (trimmedWordBeforeCursor &&
+                        trimmedWordBeforeCursor.length > 0 &&
+                        trimmedWordBeforeCursor.split(' ').length <= 4)
+                ) {
+                    this.dataProvider.queryStringUpdated(
+                        trimmedWordBeforeCursor,
+                        wordBeforeCursorWithoutTriggerChar == trimmedWordBeforeCursor
+                    );
+                    this.setLastKnownRange(this.editor?.getSelectionRange() ?? null);
```

Comment:
Suggest do a null check for this.editor at the beginning of this function so that no need to check editor everywhere.